static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -38,40 +39,48 @@ function isIssueContext(
   );
 }
 
+// NOTE: Will add release column as an eligible column.
 export function hasContext(
   dataRow: TableDataRow,
   column: TableColumn<keyof TableDataRow>
 ): boolean {
   return isIssueContext(dataRow, column);
 }
 
-// NOTE: Will extend when we add more type of contexts.
-function getUrl(dataRow: TableDataRow, column: TableColumn<keyof TableDataRow>): string {
-  return isIssueContext(dataRow, column) ? `/issues/${dataRow['issue.id']}/` : '';
-}
+type RequestParams = {
+  path: string;
+  options?: RequestOptions;
+};
 
-function fetchData(
-  api: Client,
+// NOTE: Will extend when we add more type of contexts. Context is only relevant to issue and release columns for now.
+function getRequestParams(
   dataRow: TableDataRow,
-  column: TableColumn<keyof TableDataRow>
-): Promise<Group> {
-  const promise: Promise<Group> = api.requestPromise(getUrl(dataRow, column), {
-    method: 'GET',
-    query: {
-      collapse: 'release',
-      expand: 'inbox',
-    },
-  });
-
-  return promise;
+  column: TableColumn<keyof TableDataRow>,
+  organization?: Organization
+): RequestParams {
+  return isIssueContext(dataRow, column)
+    ? {
+        path: `/issues/${dataRow['issue.id']}/`,
+        options: {
+          method: 'GET',
+          query: {
+            collapse: 'release',
+            expand: 'inbox',
+          },
+        },
+      }
+    : {
+        path: `/organizations/${organization?.slug}/releases/${dataRow.release}/`,
```

Comment:
I know this isn't rendering on the release column yet, but I think once you have an `isReleaseContext`, you should probably change this to if statements and maybe consider a reasonable output if it's none of the contexts instead of dropping to one by default, just as a safeguard

Reply:
Yeah that's the idea! Thanks.